tsconfig.json

```diff
@@ -0,0 +1,79 @@
+    "allowJs": true,
+    "checkJs": true,
+    "baseUrl": ".",
+    "paths": {
+      "internal/*": ["./lib/internal/*"],
+      "constants": ["./lib/constants.js"],
+      "net": ["./lib/net.js"],
+      "trace_events": ["./lib/trace_events.js"],
+      "events": ["./lib/events.js"],
+      "repl": ["./lib/repl.js"],
+      "util": ["./lib/util.js"],
+      "dgram": ["./lib/dgram.js"],
+      "vm": ["./lib/vm.js"],
+      "stream": ["./lib/stream.js"],
+      "child_process": ["./lib/child_process.js"],
+      "assert": ["./lib/assert.js"],
+      "_tls_wrap": ["./lib/_tls_wrap.js"],
+      "http2": ["./lib/http2.js"],
+      "util/types": ["./lib/util/types.js"],
+      "inspector": ["./lib/inspector.js"],
+      "os": ["./lib/os.js"],
+      "_http_server": ["./lib/_http_server.js"],
+      "timers/promises": ["./lib/timers/promises.js"],
+      "console": ["./lib/console.js"],
+      "perf_hooks": ["./lib/perf_hooks.js"],
+      "readline": ["./lib/readline.js"],
+      "path/win32": ["./lib/path/win32.js"],
+      "path/posix": ["./lib/path/posix.js"],
+      "punycode": ["./lib/punycode.js"],
+      "stream/promises": ["./lib/stream/promises.js"],
+      "_http_incoming": ["./lib/_http_incoming.js"],
+      "https": ["./lib/https.js"],
+      "_stream_wrap": ["./lib/_stream_wrap.js"],
+      "domain": ["./lib/domain.js"],
+      "dns": ["./lib/dns.js"],
+      "_http_client": ["./lib/_http_client.js"],
+      "diagnostics_channel": ["./lib/diagnostics_channel.js"],
+      "assert/strict": ["./lib/assert/strict.js"],
+      "tty": ["./lib/tty.js"],
+      "_http_agent": ["./lib/_http_agent.js"],
+      "timers": ["./lib/timers.js"],
+      "_http_outgoing": ["./lib/_http_outgoing.js"],
+      "querystring": ["./lib/querystring.js"],
+      "_tls_common": ["./lib/_tls_common.js"],
+      "module": ["./lib/module.js"],
+      "_stream_passthrough": ["./lib/_stream_passthrough.js"],
+      "_stream_transform": ["./lib/_stream_transform.js"],
+      "worker_threads": ["./lib/worker_threads.js"],
+      "sys": ["./lib/sys.js"],
+      "_stream_duplex": ["./lib/_stream_duplex.js"],
+      "path": ["./lib/path.js"],
+      "_http_common": ["./lib/_http_common.js"],
+      "string_decoder": ["./lib/string_decoder.js"],
+      "cluster": ["./lib/cluster.js"],
+      "v8": ["./lib/v8.js"],
+      "crypto": ["./lib/crypto.js"],
+      "wasi": ["./lib/wasi.js"],
+      "_stream_readable": ["./lib/_stream_readable.js"],
+      "zlib": ["./lib/zlib.js"],
+      "url": ["./lib/url.js"],
+      "tls": ["./lib/tls.js"],
+      "dns/promises": ["./lib/dns/promises.js"],
+      "fs/promises": ["./lib/fs/promises.js"],
+      "_stream_writable": ["./lib/_stream_writable.js"],
+      "async_hooks": ["./lib/async_hooks.js"],
+      "process": ["./lib/process.js"],
+      "http": ["./lib/http.js"],
+      "buffer": ["./lib/buffer.js"],
+      "fs": ["./lib/fs.js"],
+    },
+    "noEmit": true,
+    "target": "ESNext",
+    "module": "CommonJS"
```

Comment:
can we put this higher so it's visible before the big "paths" key?

Reply:
sure

Reply:
done